test/sqlsrv/srv_049_stream_nvarchar_utf8.phpt

```diff
@@ -0,0 +1,43 @@
+$sql = "INSERT INTO #Table VALUES (N'$data')";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Fetch
+$sql = "SELECT * FROM #Table";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Compare output
+$row = sqlsrv_fetch_array($stmt);
+echo ($row['c1'] == $data) ? "True\n" : "False\n";
```

Comment:
The main diff of this test comparing to the one above is UTF-8 charset right? Are both files saved in the same encoding?